src/engraving/rendering/dev/systemlayout.cpp

```diff
@@ -992,7 +992,10 @@ void SystemLayout::layoutSystemElements(System* system, LayoutContext& ctx)
         }
         staff_idx_t si = e->staffIdx();
         EngravingItem* parent = e->parentItem(true);
-        IF_ASSERT_FAILED(parent && parent->isSegment());
+        IF_ASSERT_FAILED(parent&& parent->isSegment())
+        {
+            ;
+        }
```

Comment:
But why that empty `IF_ASSERT_FAILED()` in the first place, when `toSegment(parent)` does the very same?
@mike-spa ?

Reply:
I think, here should be like this
```
IF_ASSERT_FAILED(parent && parent->isSegment()) {
    continue;
}
```

Reply:
Makes sense. A lot ;-)

Reply:
Hmm, maybe not, as then `toSegment(parent)` will never get called with a segment?

Reply:
@Jojo-Schmitz What do you mean? The following:
```c++
IF_ASSERT_FAILED(parent && parent->isSegment()) {
    continue;
}
```
is equivalent to 
```c++
if (!(parent && parent->isSegment()) {
    continue;
}
```
So `toSegment` will only get called with non-null segments.

Reply:
Ah, I always get confused by this inverted logic